apps/web/src/pages/DNDDigitalCareersPage/DNDDigitalCareersPage.tsx

```diff
@@ -0,0 +1,533 @@
+          {intl.formatMessage(navigationMessages.createProfile)}
+        </CTALink>
+      </SkewedImageContainer>
+      <Container className="my-18">
+        <Heading
+          color="secondary"
+          level="h2"
+          icon={IdentificationIcon}
+          className="font-normal"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Find a role that fits your skills",
+            id: "6Wl3sQ",
+            description: "Heading for finding a role with DND",
+          })}
+        </Heading>
+        <p className="my-6">
+          {intl.formatMessage({
+            defaultMessage:
+              "Check out the latest DSG opportunities in digital and tech, from entry level to management. Find a team, make a difference, and be inspired.",
+            id: "PNdaFP",
+            description: "Description about roles with DSG",
+          })}
+        </p>
+        <p className="my-6">
+          {intl.formatMessage({
+            defaultMessage:
+              "The Digital Services Group is looking for digital professionals with strongly developed skills in",
+            id: "Kg+qsx",
+            description: "Lead-in text for specific skills DSG is looking for",
+          }) + intl.formatMessage(commonMessages.dividingColon)}
+        </p>
+        <Ul space="md" className="grid max-w-md sm:grid-cols-2 sm:gap-x-6">
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "cyber security",
+              id: "Jl+oMW",
+              description: "The cyber security skill",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "systems integration",
+              id: "xdV1ZI",
+              description: "The systems integration skill",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "network security",
+              id: "f2Jqya",
+              description: "The network security skill",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "cloud computing",
+              id: "4MAklr",
+              description: "The cloud computing skill",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "software development",
+              id: "8vwKjc",
+              description: "The software development skill",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "data analytics",
+              id: "yszH29",
+              description: "The data analytics skill",
+            })}
+          </li>
+        </Ul>
+        <Note className="my-6">
+          {intl.formatMessage({
+            defaultMessage:
+              "It’s an asset if you’re familiar with military IT infrastructure and protocols, though this is not essential.",
+            id: "tQxwgs",
```

Comment:
I remember some discussion about this, what the font colour should be. Appears to currently fail contrast

<img width="900" height="98" alt="image" src="https://github.com/user-attachments/assets/e8829978-4a0e-403b-90d1-9456753a4dd0" />

Reply:
Oh weird! I swear I checked it and it was passing but you are right! I change chades and it should be passing after [fix colour contrast issue](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14556/commits/639d74c6b76796a3616ed9720983786bed306018)